codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -457,14 +457,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-deploy-plugin</artifactId>
-        <version>2.7</version>
+        <version>2.8.2</version>
         <configuration>
           <retryFailedDeploymentCount>10</retryFailedDeploymentCount>
         </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-release-plugin</artifactId>
-        <version>2.4.1</version>
+        <version>2.5.3</version>
```

Comment:
Yeah, and I had the problem with 2.5.1 last time I tried: https://issues.apache.org/jira/browse/MRELEASE-767 It's marked as resolved but was not fixed in 2.5.1. It may have been fixed in 2.5.3, but I didn't try.